crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -427,7 +427,7 @@ def get_node_and_network(node_name)
     @node = NodeObject.find_node_by_name(node_name) if @node.nil?
     if @node
       # If we're in discovery mode, then we have a temporary DHCP IP address.
-      if not ["discovering", "discovered", "hardware-installing", "hardware-installed"].include? @node.state
+      if !["discovering", "discovered"].include?(@node.state)
```

Comment:
@vuntz Does the IP the node is currently using really already match the one we allocated from the "host" range. Withouth having tested this, I'd expect the node to still use the IP address from the "dhcp" range. Displaying the allocated address in that case would be confusing IMO